canonical-websites/snapcraft.io
# Implement more complete SEO tags

# Done

Implemented the snapcrat.io meta tags et al as described in https://docs.google.com/spreadsheets/d/1Td2IxYgXpPwfQivTcKDhrx3197QQ6ZIeUq4mzXrkZqQ/edit?ts=5af2cb63#gid=0

- Added schema.org schemas, additional twitter meta tags and the og:url tag

# QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/, http://0.0.0.0:8004/store and http://0.0.0.0:8004/SNAP_NAME
- Right click and 'View source'. Ensure the meta tags are what you would expect.
